### ISSUE TITLE
ModuleNotFoundError: No module named 'xLSTM.data'

### ISSUE THREAD

**Issue by mw66** [muditbhargava66/PyxLSTM#9]
After:

```
$ pip install .
```

do

```
$ python3 examples/language_modeling.py 
...
ModuleNotFoundError: No module named 'xLSTM.data'
```

**Comment by nicolay-r** [comment]
Hi, I just come over and looking at the related script. 
I believe the issue is with the 
https://github.com/muditbhargava66/PyxLSTM/blob/a34c2526aa2646203e6d512ed3186765bbd20b3b/examples/language_modeling.py#L6

where the actulal file is stored there:
https://github.com/muditbhargava66/PyxLSTM/blob/main/data/dataset.py
So that you have to setup the correct import path, as both dataset and tokenizer are not a part of the `xLSTM` package.

**Comment by mw66** [comment]
Since it's a pip package, why not update the package to include these files to be self contained?

**Comment by nicolay-r** [comment]
I can understand your confusion, but I believe it is an Occam's razor concept. 
Dataset and Language model feature is not conceptually a part of xLSTM rather just a aprticar application. So that author mindfully not include them into the pip package.
 So that all the supplements are expected to be manually fixed.

**Comment by nicolay-r** [comment]
Simply just copy the context from the example script and update the imports

**Comment by mw66** [comment]
Then at least change 

```
from xLSTM.data import LanguageModelingDataset, Tokenizer 
```

to be

```
from data import LanguageModelingDataset, Tokenizer 
```

i.e. `data` is not part of the  `xLSTM` module.

**Comment by nicolay-r** [comment]
:ok_hand:  I believe @muditbhargava66 will back to this eventually

**Comment by muditbhargava66** [comment]
The issue has been definitively resolved in the latest update, Version 2.0.0.
